r/tests/testthat/test-Array.R

```diff
@@ -989,6 +989,75 @@ test_that("auto int64 conversion to int can be disabled (ARROW-10093)", {
   })
 })
 
+test_that("concat_arrays works", {
+  concat_empty <- concat_arrays()
+  expect_true(concat_empty$type == null())
+  expect_equal(concat_empty$length(), 0L)
```

Comment:
Thanks for testing this! It is a bit funny, but I'm sure there's some convoluted reason to need to do this!

Reply:
Aaah thanks!